themes/_core/js/product.js

```diff
@@ -84,6 +84,10 @@ function getProductUpdateUrl() {
   return dfd.promise();
 }
 
+function showErrorNextToAddtoCartButton(errorMessage) {
+  showError($('#product-availability'), errorMessage);
```

Comment:
Since the errorMessage is the same everywhere, make it optional with a default value

```
function showErrorNextToAddtoCartButton(errorMessage) {
  if (errorMessage === undefined) {
    errorMessage = 'An error occurred while processing your request';
  }

  showError($('#product-availability'), errorMessage);
```

Reply:
Change applied